ckan/lib/helpers.py

```diff
@@ -2344,6 +2344,35 @@ def mail_to(email_address, name):
     return html
 
 
+@core_helper
+def get_base_public_folder(app_conf):
```

Comment:
let's be boring and have just one way of loading our configuration. If LessCommand calls self._load_config like the other CkanCommand subclasses then it should work.